share/spice/quandl/fundamentals/fundamentals_triggers.txt

```diff
@@ -0,0 +1,54 @@
+total assets	ASSETS
```

Comment:
@brianrisk thanks for explaining.

>The code takes ordering into account to give preference to more specific phrases.

By more specific I take it you mean longer strings? I would suggest then that we alpha sort this list, and then when we slurp the file and create our trigger array, we sort the array by length.

E.g. ` @sorted = sort { length $a <=> length $b } @elements `